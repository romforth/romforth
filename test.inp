#{ifdef TESTROM
#{if step>=1
f
#}if
#{if step>=24
r
#}if
#{if step>=26
x
#}if
#{if step>=27
o
#}if
#{if step>=31
m !
#}if
#{if step==39
token 
#}if
#{if step==40
1000 
#}if
#{if step==41
1000 repl 
#}if
#{if step==42
1000 repl 
#}if
#{if step==43
1000 here 
#}if
#{if step==44
1000 here 
#}if
#{if step==45
1000 here swap 
#}if
#{if step==46
1000 here swap bl 
#}if
#{if step>=48
foo bar foo 
#}if
#}ifdef
