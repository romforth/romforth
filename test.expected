#{ifdef TESTROM
#{if step>=1
f
#}if
#{if step>=3
o
#}if
#{if step>=4
r
#}if
#{if step>=4.1
t
#}if
#{if step>=4.2
h
#}if
#{if step>=4.3
 
#}if
#{if step>=24
t
#}if
#{if step>=27
h
#}if
#{if step>=31
 rom !
#}if
#}ifdef
